android/layer/build.gradle

```diff
@@ -28,6 +28,7 @@ android {
                 arguments "-DANDROID_TOOLCHAIN=clang", "-DANDROID_STL=c++_static"
                 if (project.hasProperty("DisableOpenXR")) {
```

Comment:
Is this documented anywhere?  Like, how do I set "DisableOpenXR" and rebuild so that it is not included?

Reply:
Ugh, no.  Good idea, though.  I'll update the BUILD.md file with information on disabling the OpenXR build in the various ways.

Reply:
Added documentation.